Makefile

```diff
@@ -35,3 +35,6 @@ version:
 
 release:
 	goreleaser release --rm-dist
+
```

Comment:
need to update the version in this makefile and do a find and replace in the whole repo replacing `0.0.5` with `0.0.6`

Reply:
done